src/main/java/org/opensearch/security/user/UserService.java

```diff
@@ -243,14 +242,8 @@ public static String generatePassword() {
         CharacterRule lowercaseCharacterRule = new CharacterRule(EnglishCharacterData.LowerCase, 1);
         CharacterRule uppercaseCharacterRule = new CharacterRule(EnglishCharacterData.UpperCase, 1);
         CharacterRule numericCharacterRule = new CharacterRule(EnglishCharacterData.Digit, 1);
-        CharacterRule specialCharacterRule = new CharacterRule(EnglishCharacterData.Special, 1);
```

Comment:
[Special](https://github.com/vt-middleware/passay/blob/main/src/main/java/org/passay/EnglishCharacterData.java#L24) includes `:` symbol, but on line 305 when we get [BasicAuthToken](https://github.com/opensearch-project/OpenSearch/blob/994e11518ca0754ed6275ae7f88744b913255323/server/src/main/java/org/opensearch/identity/tokens/BasicAuthToken.java#L29) we use : as delimeter to split username and password. Hence this code block is removed.